F# 4.7 docs updates (#15674)
dotnet/docs: docs/fsharp/language-reference/computation-expressions.md

`yield!` is defined by the `YieldFrom(x)` member on the builder type, where `x` is a collection of values.

> Unlike `yield`, `yield!` must be explicitly specified - its behavior is not implicit in computation expressions.

Reviewer: ```suggestion
Unlike `yield`, `yield!` must be explicitly specified. Its behavior isn't implicit in computation expressions.
```
